### PR TITLE
Resurrect PDF/GeoPDF support

### PR DESCRIPTION
**Pull request by nirvn**
@m-kuhn , turns out the unquestionably good move to adopt vcpkg killed {georeferenced,Geo}PDF support. Crossing fingers it's not going to be a PITA to get it back.

### CODE REVIEW

File: vcpkg/overlay/qgis/vcpkg-cmake-wrapper.cmake [hunk 1]
```diff
@@ -131,6 +131,12 @@ if(QGIS_FOUND AND "@VCPKG_LIBRARY_LINKAGE@" STREQUAL "static")
   _find_and_link_library(spatialindex qgis_core spatialindex spatialindex-64)
   _find_and_link_library(freexl qgis_core)
   _find_and_link_library(qt5keychain qgis_core)
+  _find_and_link_library(poppler qgis_core)
+  _find_and_link_library(freetype qgis_core)
+  _find_and_link_library(freetype qgis_core)
+  _find_and_link_library(libbrotlicommon qgis_core)
+  _find_and_link_library(libbrotlidec qgis_core)
+  _find_and_link_library(libbrotlienc qgis_core)
```
Comment (m-kuhn) [inline vcpkg/overlay/qgis/vcpkg-cmake-wrapper.cmake:139]:
```suggestion
  _find_and_link_library(brotlidec qgis_core)
  _find_and_link_library(brotlicommon qgis_core)
```

May be better like that (80% sure)

**Comment by m-kuhn** [comment]
Do we need to copy poppler into our overlay (did we patch it)?

**Comment by nirvn** [comment]
@m-kuhn , yeah, I had to patch it. Removed the fontconfig and gperf stuff.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d10c8455bb5c368ea61fe573dd6f83c57be10cd1:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-2a549f62ec5453979a306422a7dfe76137063a31-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-2a549f62ec5453979a306422a7dfe76137063a31-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-2a549f62ec5453979a306422a7dfe76137063a31-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-2a549f62ec5453979a306422a7dfe76137063a31-x86.apk)

**Comment by nirvn** [comment]
@m-kuhn , alright, we got georeferenced PDF / GeoPDF support back on android, youpidou! 

The linux build fails, I assume it's because that isn't a static build?

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 2c93a23727707e8ee7248c32163f824992a958a9:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-e66f02dded83fdb54cd13c310072398aab1e8141-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-e66f02dded83fdb54cd13c310072398aab1e8141-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-e66f02dded83fdb54cd13c310072398aab1e8141-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2413-e66f02dded83fdb54cd13c310072398aab1e8141-x86.apk)

**Comment by nirvn** [comment]
Green status unlocked! :)

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**